src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.csproj

```diff
@@ -16,12 +16,16 @@
     <Compile Include="Microsoft.Data.SqlClient.cs" />
     <Compile Include="Microsoft.Data.SqlClient.Manual.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
-    <PackageReference Include="System.Security.Cryptography.Cng" Version="$(SystemSecurityCryptographyCngVersion)" />
+  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))">
```

Comment:
A more standard alternative to conditionally including the file in the csproj is to always include it, but use e.g. `#if NET6_0_OR_GREATER` in the source file. This also makes it more immediately clear for someone reading the file which TFMs it's relevant for.

Reply:
In general for this library we handle condition compilation in the project file where possible and avoid ifdefs in files. It can be done either way but this one follows the established pattern.